PetShelter/PetShelter/main.cpp

```diff
@@ -8,7 +8,10 @@ int main()
 	// To avoid the object get the same emotion
 	srand(time(NULL));
 
-	Dogs dog1("Siberian Husky");
-	dog1.PrintIdentity();
-	dog1.Speak();
+	//Dogs dog1("Siberian Husky");
+	//dog1.PrintIdentity();
+	//dog1.Speak();
+	
+	mainProgram a;
```

Comment:
Making a class to substitute the main generally a bad pratice, BUT if its small and related i see no problem.

C++ people say to keep things as seperately as possible, but as i said above, we dont really have to.